src/foam/nanos/theme/Theme.js

```diff
@@ -91,6 +91,12 @@ foam.CLASS({
       name: 'largeLogoEnabled',
       documentation: 'Uses largeLogo image on various views instead of logo.'
     },
+    {
+      class: 'Color',
+      name: 'backgroundLogo',
```

Comment:
Should it be called logoBackground or logoBackgrounColour?  else I would think this controls some 'background' logo.

Reply:
done